controller/lib/core/alarm.h

```diff
@@ -16,10 +16,9 @@ limitations under the License.
 #ifndef ALARM_H
 #define ALARM_H
 
-#include <stdint.h>
-
 #include "errors.h"
 #include "packet_types.h"
+#include <stdint.h>
```

Comment:
Same, why did this system header get moved below the others?

Reply:
This also matches the output of clang-format on my machine.

Reply:
Here's what's happening: clang-format is currently sorting includes, but it is not *rearranging blocks of includes*.

So if stdint.h comes before the others *and has a blank line after it* then it's left alone.  If it is in the same block as the others, it's sorted as is here.

I guess so long as we understand what's happening it's fine.  I just don't want warring formatters is all.